ch.sbb.scion.rcp.workbench.test/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,16 @@
+Manifest-Version: 1.0
+Bundle-ManifestVersion: 2
+Bundle-Name: SCION Workbench Test Fragment
+Bundle-SymbolicName: ch.sbb.scion.rcp.workbench.test;singleton:=true
+Bundle-Version: 1.0.0.qualifier
+Fragment-Host: ch.sbb.scion.rcp.workbench
+Automatic-Module-Name: ch.sbb.scion.rcp.workbench.test
+Bundle-RequiredExecutionEnvironment: JavaSE-11
+Bundle-Vendor: SBB and Contributors
+Require-Bundle: org.junit.jupiter.api,
+ org.mockito.mockito-core,
+ org.mockito.junit-jupiter,
+ wrapped.org.mockito.mockito-inline,
+ org.hamcrest,
+ net.bytebuddy.byte-buddy,
```

Comment:
Please define only the direct dependencies of the test fragment as Required Bundle and remove the rest.